openwisp_users/base/models.py

```diff
@@ -129,6 +129,25 @@ def clean(self):
         if self.phone_number == '':
             self.phone_number = None
 
+    @property
+    def permissions(self):
```

Comment:
let's add a docstring:

```python 
"""
Returns the user permissions from the cache, if the cache is
empty it will call self.get_all_permissions() and cache the result 
"""
```